xartist-frontend/src/app/explore/page.tsx

```diff
@@ -63,6 +65,17 @@ const Explore = () => {
     }
   };
 
+  // *** Debounced hover handler ***
+  const handleHover = ({ x, y }) => {
+    console.log(x, y);
+    if (hoverTimeoutRef.current) {
+      clearTimeout(hoverTimeoutRef.current);
+    }
+    hoverTimeoutRef.current = setTimeout(() => {
```

Comment:
The current implementation seems aiming to delay the converting of coordinators. However, as we've discussed yesterday, the problem that we are facing is the performance gap between the producer (hover event in our case) and consumer (flask server in our case). To address this problem, we need introduce a extra storage array to store all the coordinates generated by the hover event, then we need a scheduled timer function to pull the coordinate one by one from the array and convert it into image by calling the server.

Reply:
In other words, the handleHover function just need to store the coordinates into a array.

Reply:
Thank you